deps/compose.template

```diff
@@ -156,4 +161,7 @@ services:
       bootstrapper_l2:
         condition: service_completed_successfully
 {%- endif %}
+    depends_on:
```

Comment:
I think that the `{%- if ENABLE_BOOTSTRAPER_L2_SETUP %}` must be removed and also check that the `depends_on` section will be defined twice

Reply:
fixed!